### PR TITLE
Adding resources page

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Resources
+
+A repository for various courses, guides, blog posts that we find useful for our
+development processes.
```
Comment (frm) [inline resources/README.md]: I don't really think we should restrict this to development. Otherwise, good initiative.

**Review by frm — CHANGES_REQUESTED** [review]

File: resources/README.md (hunk 1 above)
Comment (naps62) [reply resources/README.md]: true. corrected

**Review by frm — APPROVED** [review]
I'm approving but I'd rather have it be merged when we have some more (diverse) resources. I also understand the reasons for merging right away and adding later, though.

File: resources/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Resources
+
+A repository for various courses, guides, blog posts that we find useful for our processes.
+
+## Courses
+
+### Rails Speed
+
+* [Rails Speed](https://www.railsspeed.com/): A full-stack course that gives you the tools to make Ruby on Rails applications faster, more scalable, and simpler to maintain. Look for it in Subvisual's Google Drive directory, or ask [@naps62](mailto:miguel@subvisual.co) if you need help finding it.
```
Comment (pfac) [inline resources/README.md:9]: Given that our guides are open to he public, does it make sense to have the text reference our internal Google Drive?

**Review by pfac — COMMENTED** [review]
:question:

File: resources/README.md (hunk 2 above)
Comment (naps62) [reply resources/README.md:9]: that was on purpose. Is there any harm in making it public that we have one? As long as we don't publish links to those directories (which are still protected by authentication, but still...)

Comment (pfac) [reply resources/README.md:9]: I have nothing against it, I just thought the guides were meant to be more abstract than that. Just trying to figure it out.

**Review by pfac — APPROVED** [review]

File: resources/README.md (hunk 2 above)
Comment (frm) [reply resources/README.md:9]: I agree with @pfac . This should be considered internal information. At most it should be in a handbook, not in our guides.

**Pull request closed without merging**